client/landing/stepper/declarative-flow/internals/steps-repository/subscribers/index.tsx

```
@@ -52,9 +52,9 @@ const Subscribers: Step = function ( { navigation } ) {
 							) }
 							showSubtitle={ true }
 							emailPlaceholders={ [
```

Do we need the `emailPlaceholders={}` attribute at all here? We could just let the form within onboarding default to the placeholder used inside the component? This kind of flexibility seems overly complicated for little gain.

We can then update the placeholders in the component if still need; yours seem more fun and quirky than originals.

---

👍 Done